source/server-discovery-and-monitoring/tests/integration/minPoolSize-error.yml

```diff
@@ -16,7 +16,7 @@ tests:
       configureFailPoint: failCommand
       mode: { skip: 3 }
       data:
-        failCommands: ["isMaster", "hello"]
+        failCommands: ["hello"]
```

Comment:
@JamesKovacs I think this should be `["hello", "isMaster"]` ?

Reply:
I'm going to review the integration tests tomorrow. They should include both `hello` and `isMaster` in the list of failpoints.

Reply:
Actually, it should just be `isMaster` since we want the handshake to fail.

Reply:
I think both hello and isMaster are needed because we can run these tests with apiVersion implicitly set by the test suite.

Reply:
Ah yes, you are right... without `hello`, running with versioned API will cause this test to fail.